project/Dependencies.scala

```diff
@@ -223,11 +223,15 @@ object Dependencies {
 
   val workbenchServiceTest: ModuleID = "org.broadinstitute.dsde.workbench" %% "workbench-service-test" % serviceTestV % "test" classifier "tests" excludeAll (excludeGuava, excludeStatsD)
   val leonardoClient: ModuleID =  "org.broadinstitute.dsde.workbench" %% "leonardo-client" % "1.3.6-a02f2fe-SNAP"
+  val ssh: ModuleID = "com.hierynomus" % "sshj" % "0.36.0"
+
+  val automationOverrides = List(guava)
 
   val automationDependencies = List(
     "com.fasterxml.jackson.module" %% "jackson-module-scala"   % "2.15.2" % "test",
     logbackClassic % "test",
     leonardoClient,
+    ssh,
```

Comment:
Can this be marked as a test only dependency?

Reply:
I would expect a % "test" tag if it is only for test.

Reply:
Ah, TIL that that annotation affects the classpath compilation.